canonical/ubuntu.com
# add equal employment opportunity commitment legal page

## Done

- added /legal/equal-employment-opportunity-commitment, based on [this copy doc](https://docs.google.com/document/d/1_ExS3fRc8odw8extzVeZWkrZU78DekJ3Ei_dUOayGLI/edit)
- added a card for it on /legal index.

## QA

- visit https://ubuntu-com-11925.demos.haus/legal
- see that a link to the new page appears as a card
- follow the link, see that the content of the page matches the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5787
